### PR TITLE
Provide recent applications on review page

### PR DESCRIPTION
**Pull request by lalit97**
A list of applicant's recent applications on review page will help coordinators

Bug: T186502

Screen Shot 2020-05-20 at 4 20 06 PM

### CODE REVIEW

File: TWLight/applications/views.py [hunk 1]
```diff
@@ -906,6 +907,12 @@ def get_context_data(self, **kwargs):
         if app.parent and existing_authorization:
             context["previous_auth_expiry_date"] = existing_authorization.date_expires
 
+        # Get 5 recet applications to display on application evaluation page
```
Comment (jsnshrmn) [inline TWLight/applications/views.py:910]: typo
```suggestion
        # Get 5 recent applications to display on application evaluation page
```

**Comment by lalit97** [comment]
> Thanks for your contributions! This needs a test added for the new rendered content. Just let me know if you need help with that.

I have added a test for the behaviour. It helped me to understand the task even better and I have made some changes to the query also. Sorry for the delay.

**Review by jsnshrmn — APPROVED** [review]

**Review by jsnshrmn — APPROVED** [review]
This looks great! Thanks for your contribution! I made a small update to your test after catching your branch up with master. The starting state of the editor you were using changed, so I just added a unique editor for your test.

**Comment by lalit97** [comment]
> This looks great! Thanks for your contribution! I made a small update to your test after catching your branch up with master. The starting state of the editor you were using changed, so I just added a unique editor for your test.

My pleasure :D Thanks for the review and changes you made :)

**Pull request merged**